dependencies/registry/registry.yaml

```diff
@@ -11,6 +11,9 @@ metadata:
     run: registry
   name: registry
   namespace: kind-registry
+  annotations:
+    ignore-check.kube-linter.io/no-read-only-root-fs: ""
+    ignore-check.kube-linter.io/run-as-non-root: ""
```

Comment:
Let's add here a similar comment as added to the patches created for the same annotation.

Reply:
@ShakedAviv50313  Can you address this?

Reply:
on it